docs/android/v2/release-notes/release-notes.mdx

```diff
@@ -19,6 +19,20 @@ import AndroidPrebuiltVersionShield from '@/common/android-prebuilt-version-shie
 | live.100ms:virtual-background: |<AndroidSdkVersionShield />|
 | live.100ms:hms-noise-cancellation-android: | <AndroidSdkVersionShield /> |
 
+## v2.9.74 - 2025-03-3
+### Added
+* Name space changes added 
```

Comment:
Renamed the namespace of all webrtc files, so that it does not conflict with files of any other webrtc vendor if being used in the app